doc/modules/classes.rst

```diff
@@ -1646,3 +1646,27 @@ Recently deprecated
 
 To be removed in 1.0 (renaming of 0.25)
 ---------------------------------------
+
+.. _loss_function_ref:
+
+:mod:`sklearn._loss`: Private Loss Function Classes
+===========================================================
+
```

Comment:
Transfer of discussion https://github.com/scikit-learn/scikit-learn/pull/20811#discussion_r693728993:
Quoting @NicolasHug 
> Is there a specific reason to document these?
>
>I'm not sure we should: we never expose private stuff in the API ref so far, and even though it's clear from here that these are private, it won't be as clear once users land on the individual pages of such losses. Until we have a system to visually distinguish the private stuff from the public stuff in the docs, I would recommend not to document them publicly. (related is our developer-API discussion)

@ogrisel
> Maybe we could move this to a dedicated page, somewhere under the developer documentation sections of the documentation.

Reply:
What developer docs are you referring to? IMHO these don't even qualify as relevant developer utils.

Reply:
See also https://github.com/scikit-learn/scikit-learn/pull/19088#issuecomment-812451036:
> Based on @cmarmo's suggestion in https://github.com/scikit-learn/scikit-learn/pull/19088#pullrequestreview-625124432, I've the same question: To render or not to render docstrings of this private module under API Reference?

Reply:
thanks for the refs. Let's remove them please. We don't document private implementation details, there's no reason to start now.

Reply:
Alright.